.gitignore

```diff
@@ -0,0 +1,9 @@
+# IDE - JetBrains
+.idea
+
```

Comment:
We used to reuse a pretty comprehensive gitignore for php projects, probably still exists in the older projects like sensei-import / sensei-weather.